### PR TITLE
environ configuration for variables: db_url, secret_key, time_zone, debug is done

### PR DESCRIPTION
**Pull request by victoraysev**

### CODE REVIEW

File: ecommerce/ecommerce/settings.py [hunk 1]
```diff
@@ -9,23 +9,46 @@
...
+import environ
+import os
 
-from pathlib import Path
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
+env = environ.Env(
+    # set casting, default value
+    DEBUG=(bool, False)
```
Comment (CagriYonca) [inline ecommerce/ecommerce/settings.py:18]: are you sure if debug param should be False by default?

**Review by ttoygar — APPROVED** [review]
ALLOWED_HOSTS information can also be put in the .env file. Also, due to the working principle of git diff, changing the order of variables makes it difficult to review.

**Review by VeliAliyev — APPROVED** [review]

File: ecommerce/ecommerce/settings.py (hunk 1 above)
Comment (victoraysev) [reply ecommerce/ecommerce/settings.py:18]: Yes. İt is stated in django docs that :
**Never deploy a site into production with DEBUG turned on.**.  In my opinion, default value should be false.
Thanks for review

Comment (victoraysev) [reply ecommerce/ecommerce/settings.py:18]: https://docs.djangoproject.com/en/3.2/ref/settings/

**Pull request closed without merging**